<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RetrieveDialog</class>
 <widget class="QDialog" name="RetrieveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="retrieve_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>220</width>
     <height>280</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QStackedWidget" name="ret_stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>220</width>
      <height>270</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="retrieve_page">
     <widget class="QLineEdit" name="username_edit1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="realname_edit1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="phone_edit1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="retrieve_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 11pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>找回</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="error_page">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>220</width>
        <height>240</height>
       </rect>
      </property>
      <widget class="QWidget" name="change_page">
       <widget class="QLineEdit" name="newPwd_edit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>180</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="again_edit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>180</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="error_lb">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>140</y>
          <width>220</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="change_btn">
        <property name="geometry">
         <rect>
          <x>45</x>
          <y>170</y>
          <width>130</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>确认修改</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="error_page1">
       <widget class="QLabel" name="text_lb1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>220</width>
          <height>131</height>
         </rect>
        </property>
        <property name="text">
         <string>提示</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="return_btn1">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>170</y>
          <width>140</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="retrieve_lb">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>10</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>找回密码</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="close_btn1">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
